README.md

```diff
@@ -1,47 +1,115 @@
 # Mimir bundle
 
-Mimir is a shardable, horizontally scalable push-based metrics server, and alert rule evaluation engine which is a Prometheus/Cortex/Thanos-compatible by Grafana Labs.
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [Mimir bundle](#mimir-bundle)
+    - [Usage](#usage)
+        - [Recommended or minimal deployment](#recommended-or-minimal-deployment)
+        - [Reader-Writer path deployment:](#reader-writer-path-deployment)
+    - [Overlays](#overlays)
+
+<!-- markdown-toc end -->
+
+
+
+Mimir is an open source, horizontally scalable, highly available, multi-tenant TSDB for long-term storage for Prometheus.
 
```

Comment:
I'd add a short section here explaining the broad-strokes architecture of Mimir proper, and our solution, and why they're somewhat different.
Something like

"Mimir is a single binary that can be configured to run in several modes to take up different roles in the ingestion/storage/read pipeline, such as `distributor`, `compactor` and so on. [see the official docs for more](link).

Charmed Mimir consists of two charms, a coordinator and a worker. The coordinator is responsible for A, B, C.
Several worker applications can be deployed and related to a central coordinator, taking on different roles and enabling independent scalability of the various components of the stack.

The coordinator charm acts as single access point for bundle-level integrations such as TLS, ingress, self-monitoring, etc..., and as single source of truth for bundle-level configurations that otherwise would have to be repeated (and kept in sync) between the individual worker nodes."

Reply:
(also explain perhaps the mimir 'monolithic' deployment scenario, the typical read/write distinction, and that the s3 integrator is technically only needed if you deploy mimir in non-monolithic mode.)